openpype/hosts/houdini/plugins/create/create_usd.py

```diff
@@ -16,8 +16,23 @@ class CreateUSD(plugin.HoudiniCreator):
 
     def create(self, subset_name, instance_data, pre_create_data):
 
+        # Allow creating the USD node directly in a LOP network as opposed
+        # to an /out network by defining the type based on a `parent` if
+        # it passed along.
+        node_type = "usd"
```

Comment:
should we be more explicit and just initialize this to `node_type = None` and assign `node_type = "usd"` on the `opNodeTypeCategory` condition and throw a ValueError if `node_type` is still None after?